src/wallet/rpcwallet.cpp

```diff
@@ -1695,6 +1695,11 @@ static UniValue gettransaction(const JSONRPCRequest& request)
                         {
                             {RPCResult::Type::ELISION, "", "Equivalent to the RPC decoderawtransaction method, or the RPC getrawtransaction method when `verbose` is passed."},
                         }},
+                        {RPCResult::Type::OBJ, "lastprocessedblock", "hash and height of the block this transaction information was generated on",
+                            {
+                            {RPCResult::Type::STR_HEX, "hash", "hash of the block this transaction information was generated on"},
+                            {RPCResult::Type::NUM, "height", "height of the block this transaction information was generated on"},
+                        }},
```

Comment:
Could this be made a `static const RPCResult RESULT_LAST_PROCESSED_BLOCK{RPCResult::Type::OBJ, "lastprocessedbl...` to avoid boilerplate?
```suggestion
                        RESULT_LAST_PROCESSED_BLOCK,
```